src/App.tsx

```diff
@@ -15,36 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const CLASS_IS_LIGHT = 'is-light';
+
+enum SortType {
+  BY_ALPHABET = 'alphabet',
+  BY_LENGTH = 'length',
+  BY_DEFAULT = '',
+}
+
+type GetPreparedGoods = (
+  goods: string[],
+  sortField: string,
```

Comment:
```suggestion
  sortField: SortType,
```
The type should be your enum